pkg/gator/runner_test.go

```diff
@@ -1016,6 +1016,110 @@ func TestRunner_Run(t *testing.T) {
 				}},
 			},
 		},
+		{
+			name: "admission review object",
+			suite: Suite{
+				Tests: []Test{
+					{
+						Name:       "no op admission review object", // this test is akin to a no op
```

Comment:
To be clear, this would not be akin to a no-op... this tests what the constraint template would do if it received an admission review where "object" is blank. FWIW we may want to raise an error here, as either object or oldObject should be defined to match the behavior of the K8s API server.

Reply:
This is a good point! I think it makes sense from an open box testing perspective to mimic the k8s API behavior. I added a commit to fix this!